tools/ci_build/github/azure-pipelines/py-packaging-pipeline.yml

```diff
@@ -29,6 +29,16 @@ parameters:
   type: boolean
   default: true
 
+- name: enable_olive_windows_cpu
```

Comment:
I assume you will drive this work.  I will send a list of compliance requirements to you by email. Please make a plan and let me know it.